pkgs/development/tools/vagrant/default.nix

```diff
@@ -14,8 +12,8 @@ let
     pname = "vagrant";
     inherit version;
 
-    inherit ruby;
-    gemdir = ./.;
+    gemfile = writeText "Gemfile" "";
+    lockfile = writeText "Gemfile.lock" "";
```

Comment:
I guess this works because bundler isn’t actually used at runtime?

Reply:
Either that, or it does look at runtime but it accepts empty files :man_shrugging: